ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml

```diff
@@ -8,10 +8,19 @@ import StatusQ.Controls 0.1
 import StatusQ.Components 0.1
 
 import "../../layouts"
+import "../../popups/community"
+
+import utils 1.0
 
 StackLayout {
     id: root
 
+    property bool hasAddedContacts: false
+    property var activeCommunity
+    property var communitySectionModule
```

Comment:
The backend object should not be used in the Panel (see another comment mentioning QML core architecture) and ideally, backend objects should be accessed only through stores.

Reply:
@osmaczko  I forgot, will fix